features/page_objects/front_office/registrations/confirmation_page.rb

```diff
@@ -0,0 +1,7 @@
+class ConfirmationPage < SitePrism::Page
+
+  # Registration complete
+  element(:confirmation_message, ".completeSummaryTitle")
+  element(:registration_number, "#registrationNumber")
+
+end
```

Comment:
You should remove this file :)

Reply:
Removed! I blame Git for being confusing :)